docs/platforms/dotnet/index.mdx

```diff
@@ -75,7 +78,6 @@ SentrySdk.Init(options =>
 {
     options.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.us.sentry.io/5428537";
     options.Debug = true;
-    options.AutoSessionTracking = true;
```

Comment:
Also removed this since it can't be added on a blank dotnet instruction. Otherwise this would be on by default.

On mobile for example it's already on by default. On aspnet it takes no effect since no server mode was added